opencti-platform/opencti-front/src/private/components/cases/tasks/Root.tsx

```diff
@@ -70,6 +72,7 @@ const RootTaskComponent = ({ queryRef, taskId }) => {
     [taskId],
   );
   const location = useLocation();
+  const enableReferences = useIsEnforceReference('Task') && !useGranted([BYPASSREFERENCE]);
```

Comment:
Just out of curiosity : Why did you use useIsEnforceReference only on Task and Feedbacks and not on other Root components .

Reply:
I already added useIsEnforceReference in other Root components in a previous PR, I had to add it to Task and Feedbacks also in this PR following the changes to the ContainerStixObjectsOrStixRelationships component

Reply:
Thank you ! :) 
Just a question again: I cannot set an enableReference in EntitySettings for Note/Opinion/Task. Can you explain to me why we did the change in frontend then ?

Reply:
Good point, I didn't have that in mind when calling useInEnforceReference in Note/Opinion/Task ! In theory we could just not use it, and always set `enabledReferences` to `false` for these components. But I think that it doesn't hurt to leave it like this, this way if we decide that Note/Opinion/Task can be configured with enableReference it will work without having to change it back to use useIsEnforceReference